### ISSUE TITLE
Does is support disable pan by left button of mouse?

### ISSUE THREAD

**Issue by YornQiu** [anvaka/panzoom#296]
I want to make it only pan by middle or right button

**Comment by HumboldtK** [comment]
Add this to your element oncontextmenu="return false;"

**Comment by Jacksonant** [comment]
let say i do the oncontextmenu preventdefault(), will it disable my panning action?

**Comment by Jacksonant** [comment]
> right click will popup the contextmenu, how to prevent?

Did u manage to find out the solution?

**Comment by YornQiu** [comment]
No. And sorry, I haven't used it in a while, can't give u a suggestion.
